pyserini/eval/evaluate_qa_overlap_retrieval.py

```diff
@@ -0,0 +1,329 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Most of the tokenization code here is copied from Facebook/DPR & DrQA codebase to avoid adding an extra dependency
+"""
+
+import argparse
+import copy
+import json
+import logging
+import re
+import unicodedata
+from tqdm import tqdm
+import numpy as np
+import os
+import wget
+import regex
+import collections
```

Comment:
question, are all the dependencies already part of pyserini's?

Reply:
everything was in evaluate_dpr_retrival.py except os and wget.
maybe need to add wget in requirement.txt?

Reply:
Yup, add it, please, unless you see some other workaround without.